workers/unity/Assets/Fps/Scripts/GameLogic/StateMachine/Default/DisconnectedState.cs

```diff
@@ -0,0 +1,30 @@
+namespace Fps
+{
+    public class DisconnectedState : DefaultState
+    {
+        public DisconnectedState(UIManager manager, ConnectionStateMachine owner) : base(manager, owner)
```

Comment:
Do we need to declare the empty constructor?

Reply:
yes, because we don't define a default constructor